Cython/Compiler/ExprNodes.py

```diff
@@ -723,6 +738,11 @@ def allocate_temp_result(self, code):
                 return
             self.temp_code = code.funcstate.allocate_temp(
                 type, manage_ref=self.use_managed_ref)
+
+            if self.type.is_cpp_class:
+                code.globalstate.use_utility_code(
+                    UtilityCode.load_cached("MoveIfSupported", "CppSupport.cpp")
+                    )
```

Comment:
Just wandering around, seemingly the current codebase is using hanging indent.

Reply:
Thanks - I'll fix it with my a next lot of changes. I do try to get code style right, but I'm not very good at it

Reply:
I think you guys can make use of flake8 to lint the code.  I can work on that if such change is welcome.

Reply:
There's already a code-style test in the test-suite that catches some of the more obvious mistakes. I imagine if you could add to that then it would be welcome.